backend/apps/notifications/models.py

```
@@ -32,18 +32,145 @@ class Notification(models.Model):
+
+    class Meta:
+        indexes = [
+            models.Index(
+                fields=["recipient", "channel_id", "status"],
+                name="idx_delivery_rl",
+            ),
+            models.Index(
+                fields=["notification", "channel_id"],
+                name="idx_delivery_notif_chan",
+            ),
+        ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)